frontend/src/pages/ActivityReport/index.js

```diff
@@ -1,80 +1,129 @@
-import { useForm } from 'react-hook-form';
+import { Helmet } from 'react-helmet';
+import { Button } from '@trussworks/react-uswds';
 
+import ActivitySummary from './Pages/ActivitySummary';
+import TopicsResources from './Pages/TopicsResources';
+import NextSteps from './Pages/NextSteps';
+import ReviewSubmit from './Pages/ReviewSubmit';
+import GoalsObjectives from './Pages/GoalsObjectives';
+import Navigator from '../../components/Navigator';
 import Container from '../../components/Container';
 
-import SectionOne from './SectionOne';
-import SectionTwo from './SectionTwo';
-import SectionThree from './SectionThree';
-
 import './index.css';
 
+const pages = [
+  {
+    label: 'Activity summary',
+    renderForm: (hookForm) => {
+      const {
+        register, watch, setValue, getValues, control,
+      } = hookForm;
+      return (
+        <ActivitySummary
+          register={register}
+          watch={watch}
+          setValue={setValue}
+          getValues={getValues}
+          control={control}
+        />
+      );
+    },
+  },
+  {
+    label: 'Topics and resources',
+    renderForm: (hookForm) => {
+      const { control, register } = hookForm;
+      return (
+        <TopicsResources
+          register={register}
+          control={control}
+        />
+      );
+    },
+  },
+  {
+    label: 'Goals and objectives',
+    renderForm: () => (
+      <GoalsObjectives />
+    ),
+  },
+  {
+    label: 'Next steps',
+    renderForm: () => (
+      <NextSteps />
+    ),
+  },
+  {
+    label: 'Review and submit',
+    renderForm: () => (
+      <ReviewSubmit />
+    ),
+  },
+];
+
+const defaultValues = {
+  'activity-method': [],
+  'activity-type': [],
+  attachments: [],
+  cdi: '',
+  duration: '',
+  'end-date': null,
+  grantees: [],
+  'number-of-participants': '',
+  'other-users': [],
+  'participant-category': '',
+  participants: [],
+  'program-types': [],
+  reason: [],
+  requester: '',
+  'resources-used': '',
+  'start-date': null,
+  'target-populations': [],
+  topics: [],
+};
+
 function ActivityReport({ initialData }) {
-  const [data, updateData] = useState();
-  const hookForm = useForm({
-    mode: 'all',
-    defaultValues: initialData,
-  });
+  const [submitted, updateSubmitted] = useState(false);
 
-  const onSubmit = (formData) => {
-    // Console logging form data on submit until we can send the data to
-    // the api
+  const onFormSubmit = (data) => {
     // eslint-disable-next-line no-console
-    console.log(formData);
-    updateData(formData);
+    console.log('Submit form data', data);
+    updateSubmitted(true);
   };
 
-  const {
-    register,
-    watch,
-    setValue,
-    control,
-    formState,
-    handleSubmit,
-    getValues,
-  } = hookForm;
+  const resetForm = () => {
+    updateSubmitted(false);
+  };
 
   return (
     <>
+      <Helmet titleTemplate="TTA Smart Hub - Activity Report - %s" defaultTitle="TTA Smart Hub - Activity Report" />
```

Comment:
Would it be better to use a template of "%s - Activity Report - TTA Smart Hub"? At least in my browser, the actual page you're on ends up not fitting in the visible part of the tab

Reply:
Yeah seems like most of the time you want the title to be most specific to lest specific (github titles this page as `Add navigator by jasalisbury - Pull Request #190 - HHS/Head-Start-TTADP`). I'll update.